docs/basics.md

```diff
@@ -471,8 +471,16 @@ To see how the test was executed, use [stepByStepReport Plugin](/plugins/#stepby
 
 ## Retries
 
+### Auto Retry
+
+You can auto-retry a failed step by enabling [retryFailedStep Plugin](/plugins/#retryfailedstep).
+
+> **autoRetry plugin is enabled by default** for new setups since CodeceptJS 2.4
```

Comment:
Why this differs from 463, 453 rows?

Do we need badges or quoted messages?

Reply:
Yeah, added quotes